tests/stub/driver_execute_query/scripts/transaction_chaining.script

```diff
@@ -0,0 +1,38 @@
+!: BOLT 5.0
+!: ALLOW CONCURRENT
+
+A: HELLO {"{}": "*"}
+*: RESET
+{*
+    {{
+        C: BEGIN {"[db]": "*"}
+        S: SUCCESS {}
+        {{
+            C: RUN "CREATE (p:Person{name:$name}) RETURN p.name AS name" { "name":"the person" } {}
```

Comment:
can we make the query simpler? does it need to be that long? (applies to other places too)

Reply:
I made this way for making it easier to figure out what's going on.